settings/site.ini

```diff
@@ -563,6 +563,7 @@ UserCreatorID=14
 # md5_site generates password hash from site, user and password
 # plaintext does not generate a hash but has the password as it is, this is not
 # recommended since it is a huge security risc.
+# bcrypt generates password bcrypt from password only.
 # note: password hashes generated with md5_site will not work after
 #       changing the site name.
 HashType=md5_user
```

Comment:
So why not change the default hash type to bcrypt, like in https://github.com/ezsystems/ezpublish-kernel/pull/1388 ?